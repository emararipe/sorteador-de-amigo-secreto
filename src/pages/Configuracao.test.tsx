import Configuracao from './Configuracao'
import { RecoilRoot } from 'recoil'
import { render } from '@testing-library/react'

const mockNavigate = jest.fn()

jest.mock('react-router-dom', () => {
  return {
    useNavigate: () => mockNavigate
  }
})

describe('a pagina de configuracao', () => {
  test('deve ser renderizada corretamente', () => {
      const { container } = render(<RecoilRoot>
          <Configuracao />
      </RecoilRoot>)

      expect(container).toMatchSnapshot()
  })
})
