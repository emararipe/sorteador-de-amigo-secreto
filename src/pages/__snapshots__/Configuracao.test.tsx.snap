// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a pagina de configuracao deve ser renderizada corretamente 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Vamos come√ßar!
      </h2>
      <form>
        <input
          placeholder="Insira os nomes dos participantes"
          type="text"
          value=""
        />
        <button
          disabled=""
        >
          Adicionar
        </button>
        
      </form>
      <ul />
      <footer>
        <button
          disabled=""
        >
          Iniciar a brincadeira
        </button>
      </footer>
    </section>
  </div>
</div>
`;
